### ISSUE TITLE
SENSITIVE-DATA and gid:138

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ype/pulledpork#23]
```

The current version of PP does not support disabling/enabling gid's other
than 1 and 3 in disablesid.conf and enablesid.conf. Need to at lease add
gid 138 due to the new sensitive data rules.
```

Original issue reported on code.google.com by `Jason.R....@gmail.com` on 13 May 2010 at 4:51

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed in 0.4.2 (currently in SVN).  The fix allows for any GID to 
be 
modifiable.
```

Original comment by `Cummin...@gmail.com` on 13 May 2010 at 4:54
- Changed state: **Fixed**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I still am unable to disable to sensitive data rules via disabledsid.conf Can 
you give me a hand?

Tj
```

Original comment by `t...@valleynerds.com` on 5 Feb 2013 at 9:32
